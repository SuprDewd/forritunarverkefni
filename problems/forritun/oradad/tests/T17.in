100 -94 -9 -83
