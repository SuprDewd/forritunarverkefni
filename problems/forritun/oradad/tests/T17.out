-83 -94 -9 100
